diff --git a/experimental/libbox/http.go b/experimental/libbox/http.go
index 000000000..000000000 100644
--- a/experimental/libbox/http.go
+++ b/experimental/libbox/http.go
@@ -56,6 +56,7 @@ type HTTPRequest interface {
 
 type HTTPResponse interface {
 	GetContent() (*StringBox, error)
+	GetHeader(key string) string
 	WriteTo(path string) error
 	WriteToWithProgress(path string, handler HTTPResponseWriteToProgressHandler) error
 }
@@ -239,6 +240,10 @@ func (h *httpResponse) GetContent() (*StringBox, error) {
 	return wrapString(string(h.content)), nil
 }
 
+func (h *httpResponse) GetHeader(key string) string {
+	return h.Header.Get(key)
+}
+
 func (h *httpResponse) WriteTo(path string) error {
 	defer h.Body.Close()
 	file, err := os.Create(path)
